selfdrive/boardd/panda.h

```diff
@@ -44,19 +44,24 @@ class Panda {
  private:
   libusb_context *ctx = NULL;
   libusb_device_handle *dev_handle = NULL;
+  libusb_device **dev_list = NULL;
   std::mutex usb_lock;
   void handle_usb_issue(int err, const char func[]);
   void cleanup();
 
  public:
-  Panda();
+  Panda(std::string serial);
   ~Panda();
 
+  std::string usb_serial;
```

Comment:
make this a getSerial function and read like the python lib does https://github.com/commaai/panda/blob/master/python/__init__.py#L431

Reply:
we already have get_serial function. And this is usb_serial, it is different from one returned by get_serial() in C++ or Python function.